react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+class Stopwatch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      player: true,
+      timer: 0,
+      myVar: ''
+    };
+    this.handlePlayer = this.handlePlayer.bind(this);
+    this.handleStop = this.handleStop.bind(this);
+    this.handleClear = this.handleClear.bind(this);
+  }
+
+  render() {
+    const Player = this.state.player;
+    if (Player) {
+      return (
+        <div className="container">
+          <div onClick = {this.handleClear} className="circle">
+            <p>{this.state.timer}</p>
+          </div>
+          <i onClick = {this.handlePlayer} className="fas fa-play"></i>
+        </div>
+      );
+    }
+    return (
+      <div className="container">
+        <div onClick = {this.handleClear} className="circle">
+          <p>{this.state.timer}</p>
+        </div>
+        <i onClick = {this.handleStop} className="fas fa-pause"></i>
+      </div>
+    );
```

Comment:
Here we want to avoid duplicating the JSX, it would be better to use the `if` statements to determine which classNames and methods need to be used then having a single `return` at the end.